scripts/iSnobal/output_pngs.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+"""This script generates PNG files from a list of NetCDF files created from daily iSnobal runs and creates a GIF from the PNG files."""
+
+import xarray as xr
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np
+import pathlib
+
+def parse_args():
+    """
+    Parse command line arguments.
+
+    Returns:
+        argparse.Namespace: Parsed arguments.
+    """
+    parser = argparse.ArgumentParser(description='Generate pngs from netcdf files')
+    parser.add_argument('workdir', type=str, default="./", help='working directory')
+    return parser.parse_args()
+
+def fn_list(thisDir, fn_pattern, verbose=False):
+    """
+    Match and sort filenames based on a regex pattern in specified directory.
+
+    Parameters:
+        thisDir (str): Directory path to search.
+        fn_pattern (str): Regex pattern to match files.
+        verbose (bool): Print filenames.
+
+    Returns:
+        list: List of filenames matched and sorted.
+    """
+    import glob
```

Comment:
I am a fan declaring all imports at the top. Gives a new user a better idea on what this script is dependent on

Reply:
Noted and moved!